sellercenter-seller/src/main/java/com/gfg/sellercenter/seller/SellerReaderService.java

```diff
@@ -0,0 +1,51 @@
+package com.gfg.sellercenter.seller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.gfg.sellercenter.seller.infra.JsonHttpReader;
+import lombok.AllArgsConstructor;
+import org.json.JSONArray;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@AllArgsConstructor
+public class SellerReaderService implements SellerReaderServiceInterface {
+    @NotNull
+    @NotEmpty
+    private final String hostUrl;
+
+    @NotNull
+    private final JsonHttpReader jsonReader;
+
+    private static final String API_PATH = "/api/sellers/v1";
+
+    @Override
+    public Map<Integer, Seller> getByIds(List<Integer> sellerIds) throws IOException {
```

Comment:
We also need a method which would read only 1 seller by id

Reply:
Added this method as well